org.eclipse.dartboard/plugin.xml

```diff
@@ -174,7 +174,7 @@
        <launchGroup
              bannerImage="platform:/plugin/org.eclipse.debug.ui/icons/full/wizban/run_wiz.png"
              id="org.eclipse.dartboard.launchGroup"
-             image="platform:/plugin/org.eclipse.debug.ui/icons/full/etool16/run_exc.png"
+             image="assets/img/dart_16.png"
              label="Dart Launch Group"
              mode="run">
        </launchGroup>
```

Comment:
One other place is  
org.eclipse.dartboard.launch.LaunchConfigTab.LaunchConfigTab()
		ImageDescriptor descriptor = AbstractUIPlugin.imageDescriptorFromPlugin("org.eclipse.pde.ui", //$NON-NLS-1$
				"/icons/obj16/run_exc.png"); //$NON-NLS-1$
		image = descriptor != null ? descriptor.createImage() : null;

Reply:
Done.